database/tbcd/level/level.go

```diff
@@ -146,10 +158,14 @@ func New(ctx context.Context, cfg *Config) (*ldb, error) {
 	} else {
 		log.Infof("block cache: DISABLED")
 	}
-	if cfg.BlockheaderCache > 0 {
-		l.headerCache = lowIQMapNew(cfg.BlockheaderCache)
-
-		log.Infof("blockheader cache: %v", cfg.BlockheaderCache)
+	if cfg.blockheaderCacheSize > 0 {
+		l.headerCache, err = lowIQMapNewSize(cfg.blockheaderCacheSize)
+		if err != nil {
+			panic(spew.Sdump(cfg))
```

Comment:
Was this added for debugging purposes? If so, can it be removed?
```suggestion
```

Reply:
Yeah, just wanted early eyes on this before moving forward.